python/ray/util/tests/test_metrics.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
We need to add it to bazel build file so the test will run.

Concretely, add it to `python/ray/util/BUILD`. You can check `python/ray/dag/BUILD` as a reference.

We also need to add to `core.rayci.yml`.

If you are ok, I can also change and push to your branch.